.travis.yml

```diff
@@ -12,6 +15,7 @@ script:
   - ./plugin/gradlew -p ./plugin build --no-daemon
   - ./plugin/gradlew -p ./plugin ktlintCheck --no-daemon
   - ./gradlew ktlintCheck --no-daemon
+  - ./gradlew addKtlintCheckGitPreCommitHook --no-daemon && checkbashisms .git/hooks/pre-commit
```

Comment:
Just sanity checking `checkbashisms .git/hooks/pre-commit` will return a non-zero exit code if this fails thus failing Travis CI right?

Reply:
according to man page:
```
EXIT VALUES
       The exit value will be 0 if no possible bashisms or other problems were detected.  Otherwise it will be the sum of the following error values:

       1      A possible bashism was detected.

       2      A file was skipped for some reason, for example, because it was unreadable or not found.  The warning message will give details.

       4      No bashisms were detected in a bash script.
```